src/pattern.rs

```diff
@@ -13,6 +13,35 @@
 //! * `%T` - The name of the thread that the log message came from.
 //! * `%t` - The target of the log message.
 //!
+//! # Color Codes
+//!
+//! When logging to console, color codes can be used to display text in different colors.
+//!
+//! The basic syntax looks like this:
+//! `%c:{color}(...)` , where {color} is one of the following:
```

Comment:
It seems like it would make more sense to use `{}` instead of `()` to stay in line with `%d` formats?